server/src/main/java/org/apache/druid/catalog/model/table/InputFormats.java

```diff
@@ -127,8 +130,8 @@ public FlatTextFormatDefn(
           typeTag,
           CatalogUtils.concatLists(
               Arrays.asList(
-                  new StringPropertyDefn(LIST_DELIMITER_PROPERTY),
-                  new IntPropertyDefn(SKIP_ROWS_PROPERTY)
+                  new StringPropertyDefn(LIST_DELIMITER_PROPERTY, PropertyAttributes.OPTIONAL_SQL_FN_PARAM),
```

Comment:
Question: What are the default values of these properties if not given, and where are the defaults defined?